iothub_client/tests/common_longhaul/iothub_client_statistics.c

```diff
@@ -1460,6 +1460,11 @@ int iothub_client_statistics_get_device_twin_reported_summary(IOTHUB_CLIENT_STAT
 
                 summary->updates_received = summary->updates_received + 1;
             }
+            else if (device_twin_info->send_status_code == 408)  // http status 408 = Request Timeout
+            {
+                summary->updates_sent--;
+                LogInfo("Removing twin reported update id (%d) because of [(408) Request Timeout] network error", (int)device_twin_info->update_id);
```

Comment:
```suggestion
                LogInfo("Removing twin reported update id (%d) because of [(408) Request Timeout] internal server error", (int)device_twin_info->update_id);
```

Reply:
Since the device wasn't disconnected (that's the case handled below) and we got a server response error code, I assume the time-out is the server, not the device->FE connection timeout.

Can we also print other error details such as trackingID (if not already logged) in case the service team wants to investigate?

Reply:
No, this is coming from the http REST service call. When this happen, I see also network disconnect on the device logs around the same time, so most likely an Azure network infra issue. The service calls does not have a persistent connection, so no reasonable way to track that.